packages/swap/src/utils/rpc.ts

```diff
@@ -18,20 +13,11 @@ const readAssetValue = (
   return chainMinimums[asset.asset as keyof typeof chainMinimums];
 };
 
-export const getMinimumDepositAmount = async (
```

Comment:
not used anymore? 🤔

Reply:
was never used after a refactor in the original PR